docs/user-guides/client.md

```diff
@@ -334,6 +334,85 @@ Finally, in the return you can observe the matches are re-ranked according to `.
+
+```python
+from clip_client import Client
+from docarray import Document
+
+c = Client('grpc://0.0.0.0:23456')
+
+da = [
+    Document(text='she smiled, with pain'),
+    Document(uri='apple.png'),
+    Document(uri='apple.png').load_uri_to_image_tensor(),
+    Document(blob=open('apple.png', 'rb').read()),
+    Document(uri='https://clip-as-service.jina.ai/_static/favicon.png'),
+    Document(
+        uri='data:image/gif;base64,R0lGODlhEAAQAMQAAORHHOVSKudfOulrSOp3WOyDZu6QdvCchPGolfO0o/XBs/fNwfjZ0frl3/zy7////wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAkAABAALAAAAAAQABAAAAVVICSOZGlCQAosJ6mu7fiyZeKqNKToQGDsM8hBADgUXoGAiqhSvp5QAnQKGIgUhwFUYLCVDFCrKUE1lBavAViFIDlTImbKC5Gm2hB0SlBCBMQiB0UjIQA7'
+    ),
+]
+
+r = c.index(da)
+```
+Now that the return result is a DocumentArray, we can get a summary of it.
+
+```text
+╭──────────────────────────── Documents Summary ─────────────────────────────╮
+│                                                                            │
+│   Length                        6                                          │
+│   Homogenous Documents          False                                      │
+│   4 Documents have attributes   ('id', 'mime_type', 'uri', 'embedding')    │
+│   1 Document has attributes     ('id', 'mime_type', 'text', 'embedding')   │
+│   1 Document has attributes     ('id', 'embedding')                        │
+│                                                                            │
+╰────────────────────────────────────────────────────────────────────────────╯
+╭────────────────────── Attributes Summary ───────────────────────╮
+│                                                                 │
+│   Attribute   Data type      #Unique values   Has empty value   │
+│  ─────────────────────────────────────────────────────────────  │
+│   embedding   ('ndarray',)   6                False             │
+│   id          ('str',)       6                False             │
+│   mime_type   ('str',)       5                False             │
+│   text        ('str',)       2                False             │
+│   uri         ('str',)       4                False             │
+│                                                                 │
+╰─────────────────────────────────────────────────────────────────╯
+```
+
+The `embedding` is the output of the encoder, which is a 512-dim vector. 
+Now we can use the indexer to search for the indexed Documents.
+
+
+(searching)=
+## Searching
+
+You can use {func}`~clip_client.client.Client.search` or {func}`~clip_client.client.Client.asearch`
+to search for relevant Documents in the index for a given query.
+
+```python
+from clip_client import Client
+
+c = Client('grpc://0.0.0.0:23456')
+
+result = c.search(['smile'])
+```
+
+
+The results will look like this, the most relevant doc is "she smiled, with pain" with the cosine distance of 0.096. And the apple image has the cosine distance of 0.799.
```

Comment:
I think we can explain a little bit about how to interpret the result. For example: 'since by default we use cosine score, the smaller the value means...'